ccd-definition/CaseEvent/CareSupervision/StateChange.json

```diff
@@ -30,7 +30,7 @@
     "PostConditionState": "Deleted",
     "SecurityClassification": "Public",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/case-deletion/about-to-submit",
-    "ShowSummary": "N",
+    "ShowSummary": "Y",
```

Comment:
Is this one required? All that is shown is this 
![image](https://user-images.githubusercontent.com/22620804/88921618-bac21480-d266-11ea-9759-4f1e1a01ce45.png)
and I am not sure that is the most useful